crates/sui-graphql-rpc/src/types/sui_address.rs

```diff
@@ -87,6 +89,12 @@ impl TryFrom<Vec<u8>> for SuiAddress {
     }
 }
 
+impl From<SuiAddress> for ObjectID {
```

Comment:
nit: Could you put this next to (below) the `From<ObjectID> for SuiAddress` impl?